Gulpfile.js

```diff
@@ -40,3 +41,39 @@ gulp.task('watch', function () {
     browserSync.reload();
   });
 });
+
+
+
+
+var gulpSSH = new GulpSSH({
+  ignoreErrors: false,
+  sshConfig: {
+    host: process.env.DEPLOY_HOST,
+    username: process.env.DEPLOY_USER,
+    privateKey: process.env.DEPLOY_KEY
+  }
+});
+
+gulp.task('deploy', [
+  'css',
+  'js',
+  'deploy:assets',
+  'deploy:content',
+  'deploy:site'
+]);
+
+
+gulp.task('deploy:assets', function () {
+  return gulp.src('assets/**/*')
+      .pipe(gulpSSH.dest(process.env.DEPLOY_DIR + '/assets'));
+});
+
+gulp.task('deploy:content', function () {
+  return gulp.src('content/**/*')
+      .pipe(gulpSSH.dest(process.env.DEPLOY_DIR + '/content'));
+});
+
+gulp.task('deploy:site', function () {
+  return gulp.src('site/**/*')
+      .pipe(gulpSSH.dest(process.env.DEPLOY_DIR + '/site'));
+});
```

Comment:
Probably this, although I did not test it. Works with copying the directories:

```
gulp.task('deploy', function () {
  return gulp.src('{assets,content,site}/**/*')
      .pipe(gulpSSH.dest(process.env.DEPLOY_DIR));
});
```

Not sure whether this takes the three directories or their contents. Give this a try though.